.run/Get Upgrade Phases.run.xml

```diff
@@ -0,0 +1,5 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="Get Upgrade Phases" type="NodeJSConfigurationType" application-parameters="examples/platformvm/getUpgradePhases.ts" path-to-node="$USER_HOME$/.nvm/versions/node/v16.20.2/bin/node" node-parameters="-r tsconfig-paths/register" path-to-js-file="node_modules/ts-node/dist/bin.js" working-dir="$PROJECT_DIR$">
```

Comment:
what's this file used for ? are we still using node 16 in caminojs ?

Reply:
Otherwise, the examples don't run - I can adjust the examples to be compatible with higher versions.

Reply:
if its ide specific, it should be gitignored

Reply:
Gitignored. In the next release, I will **increase the node version required to run the examples** and add to README.md how to set up run files (to **enable examples debugging**) via IDE-specific run files.